app/controllers/field_test/experiments_controller.rb

```diff
@@ -0,0 +1,16 @@
+# This controller originally came from
+# https://github.com/ankane/field_test/blob/master/app/controllers/field_test/experiments_controller.rb
```

Comment:
Ok, I can deal with that: I do think we're potentially complicating any future upgrades if FieldTest makes big changes on their end, but the surface area of this 'overwrite' isn't so big that it would be a blocker.